app/routers/world_clock.py

```diff
@@ -0,0 +1,744 @@
+    'Pacific/Gambier': {'country': 'Australia',
+                        'subcountry': 'South Australia'},
+    'Pacific/Guam': {'country': 'Brazil', 'subcountry': 'Pará'},
+    'Pacific/Honolulu': {'country': 'United States',
+                         'subcountry': 'Hawaii'},
+    'Pacific/Majuro': {'country': 'Marshall Islands',
+                       'subcountry': 'Majuro Atoll'},
+    'Pacific/Norfolk': {'country': 'Canada', 'subcountry': 'Ontario'},
+    'Pacific/Pago_Pago': {'country': 'American Samoa',
+                          'subcountry': 'Eastern District'},
+    'Pacific/Palau': {'country': 'Mexico', 'subcountry': 'Coahuila'},
+    'Pacific/Port_Moresby': {'country': 'Papua New Guinea',
+                             'subcountry': 'National Capital'},
+    'Pacific/Tarawa': {'country': 'Kiribati',
+                       'subcountry': 'Gilbert Islands'},
+    'Pacific/Wake': {'country': 'United Kingdom', 'subcountry': 'England'},
+}
+CONTINENTS = ['Africa', 'America', 'Antarctica', 'Asia',
+              'Australia', 'Europe', 'Indian', 'Pacific']
+CITY_NAME_KEY = "name"
+COUNTRY_NAME_KEY_IN_CONTITNENTS = "Country_Name"
+COUNTRY_NAME_KEY_IN_CITIES = "country"
+SUBCOUNTRY_NAME_KEY_IN_CITIES = "subcountry"
+CONTINENT_NAME_KEY = "Continent_Name"
+CONTINENT = 'continent'
+COUNTRY = 'country'
+PLACE = 'place'
+CURRENT_TIME_KEY = 'datetime'
+TIMEZONE_PARTS_MAP = {CONTINENT: 0, PLACE: -1}
+TIME_SCOPE_INDEX = 0
+FEEDBACK_INDEX = 1
+ARB_COUNTRY_TIMEZONE = -1
+PATH_SEPARETOR = '/'
+PARTS_OF_THE_DAY_FEEDBACK = {'Early morning':
+                             [('05:00:00', '07:59:59'), 'Better not'],
+                             'Morning':
+                             [('08:00:00', '10:59:59'), 'OK'],
+                             'Late morning':
+                             [('11:00:00', '11:59:59'), 'OK'],
+                             'Early afternoon':
+                             [('12:00:00', '12:59:59'), 'OK'],
+                             'Afternoon':
+                             [('13:00:00', '15:59:59'), 'OK'],
+                             'Late afternoon':
+                             [('16:00:00', '16:59:59'),
+                              'Can be considered'],
+                             'Early evening':
+                             [('17:00:00', '18:59:59'),
+                              'Can be considered'],
+                             'Evening':
+                             [('19:00:00', '20:59:59'), 'Better not'],
+                             'Night':
+                             [('21:00:00', '23:59:59'), 'Better not'],
+                             'Late night':
+                             [('00:00:00', '04:59:59'),
+                              'Not possible'], }
+
+
+def normalize_continent_name(continent_name: str) -> Optional[str]:
+    """Normalize a given continent name to the continent name
+       that appears in the timezone list.
+
+
```

Comment:
Only one line of spacing is required here

Reply:
I didn't saw an extra spacing. I think it is now OK.